docs/developer-guide/etendo-classic/bundles/platform/etendo-webhooks.md

```diff
@@ -47,13 +47,19 @@ The access tab allows you to create access that will be used in the URL call.
 | Is required     | Whether the parameter is required or not                   |
 | Active          | Parameter status (Active by default)                       |
 
-#### Access
+#### Token Access
+
+Allow execution via token.
 
 | Field           | Description                                                                                 |
 |-----------------|---------------------------------------------------------------------------------------------|
 | Active          | Access status (Active by default)                                                           |
 | Token           | Selector with the token that will be used in the URL call created on User API Token window  |
 
+#### Secure web services (SWS) Access
+
+If you need to allow authenticated users with SWS read the guide: [How to use secure web services](../../how-to-guides/how-to-use-secure-webservices). For this you have to add roles that are allowed to execute a webhook in the "Roles" tab.
```

Comment:
```suggestion
If you need to allow authenticated users with SWS read the guide: [How to use secure web services](../../how-to-guides/how-to-use-secure-webservices.md). For this you have to add roles that are allowed to execute a webhook in the "Roles" tab.
```

Reply:
Anyway it is not clear to me, the tab roles in which window is it? Can you add a screenshot?